lib/backend/services/hive_service.dart

```diff
@@ -134,11 +132,10 @@ class HiveCache implements LocalCache {
 
     try {
       if (Hive.isBoxOpen(HiveBoxes.kPack)) {
-        final pack = await Hive.lazyBox<ExpressionResponse>(HiveBoxes.kPack);
+        final pack = await Hive.box<ExpressionResponse>(HiveBoxes.kPack);
         await pack.deleteAll(pack.keys);
       } else {
-        final pack =
-            await Hive.openLazyBox<ExpressionResponse>(HiveBoxes.kPack);
+        final pack = await Hive.openBox<ExpressionResponse>(HiveBoxes.kPack);
```

Comment:
Maybe `openLazyBox` should be used here. In cases where the box is already opened, the instance is used.

Reply:
I'll check that